Services/Services_bases/webhook_service/webhook.py

```diff
@@ -23,29 +23,40 @@
 import pyngrok.exception
 
 import octobot_commons.logging as bot_logging
+import octobot_commons.configuration as configuration
 import octobot_services.constants as services_constants
 import octobot_services.services as services
 import octobot.constants as constants
 
 
 class WebHookService(services.AbstractService):
     CONNECTION_TIMEOUT = 3
+    CONFIG_NGROK = "ngrok"
+    CONFIG_WEBHOOK_SERVER_IP = "webhook-bind-ip"
+    CONFIG_WEBHOOK_SERVER_PORT = "webhook-bind-port"
     LOGGERS = ["pyngrok.ngrok", "werkzeug"]
 
     def get_fields_description(self):
         return {
-            services_constants.CONFIG_NGROK_TOKEN: "The ngrok token used to expose the webhook to the internet."
+            WebHookService.CONFIG_NGROK: "Use Ngrok",
```

Comment:
Can you add these constants in OctoBot-Services ?

Reply:
yes, https://github.com/Drakkar-Software/OctoBot-Services/pull/103